public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,82 @@
+# Setup Google Drive remote
+
+To setup Google Drive remote you should
+[create and configure Google Project](#create-and-configure-google-project) if
+you don't have one yet.
+
+## Create and Configure Google Project
+
+Since Google Drive has tight API usage quotas, creation and configuration of
```

Comment:
let's mention also that this is required once, only one pair secret_id, client_id are required and could be share inside team safely